opendbc/safety/tests/libsafety/SConscript

```diff
@@ -30,6 +30,17 @@ env = Environment(
 if system == "Darwin":
   env.PrependENVPath('PATH', '/opt/homebrew/bin')
 
+try:
+  if GetOption('unoptimized'):
+    flags = [
```

Comment:
can we just always use these options? does it make the tests meaningfully slower?

Reply:
Let me undo it and we see how it impacts the pipeline. As long as this is only done while on PC (not on tizi) we should be fine. Otherwise we'd appreciate the optimizations :D